be/src/storage/kv_store.cpp

```diff
@@ -80,6 +82,28 @@ KVStore::~KVStore() {
     }
 }
 
+int64_t KVStore::calc_rocksdb_write_buffer_size() {
+    // 1. Get the number of disks
+    std::vector<starrocks::StorePath> paths;
+    parse_conf_store_paths(config::storage_root_path, &paths);
+    int64_t disk_cnt = paths.size();
+    // 2. Get the total memory bytes of BE
+    int64_t total_mem_bytes = MemInfo::physical_mem();
```

Comment:
better to use process_mem_limit() instead of MemInfo::physical_mem

Reply:
@trueeyu exec_env will be init after kvstore init...

Reply:
So I don't think we can use that.

Reply:
then it will be problematic in case of hybrid deployment in a single large VM. the mem_limit can be half of the physical mem. Possibly taking config::mem_limit into consideration here.

Reply:
Sorry, I think i mixup the `GlobalEnv` and `exec_env`, process mem tracker will init in `GlobalEnv::init`, and `GlobalEnv::init` will happen before kvstore init, so I think I can use `process_mem_limit` instead of `physical_mem`.

Reply:
done